src/main/java/com/linkedin/xinfra/monitor/services/ConsumeService.java

```diff
@@ -231,21 +230,13 @@ public synchronized void start() {
       _consumeThread.start();
       LOG.info("{}/ConsumeService started.", _name);
 
-      Sensor topicPartitionCount = metrics.sensor("topic-partitions");
-      DescribeTopicsResult describeTopicsResult = _adminClient.describeTopics(Collections.singleton(_topic));
-      Map<String, KafkaFuture<TopicDescription>> topicResultValues = describeTopicsResult.values();
-      KafkaFuture<TopicDescription> topicDescriptionKafkaFuture = topicResultValues.get(_topic);
-      TopicDescription topicDescription = null;
-      try {
-        topicDescription = topicDescriptionKafkaFuture.get();
-      } catch (InterruptedException | ExecutionException e) {
-        LOG.error("Exception occurred while getting the topicDescriptionKafkaFuture for topic: {}", _topic, e);
-      }
+      TopicDescription topicDescription = getTopicDescription(_adminClient, _topic);
```

Comment:
Should we move `_adminClient` and `_topic` to AbstractService and pass no parameter to this method? (similar for ProduceService)